containers/runtime/xds/config/README.md

```diff
@@ -0,0 +1,37 @@
+# Configuration for xDS server
```

Comment:
Would you mind running `prettier --write --print-width=80 --prose-wrap=always` on documentation files?

Reply:
I will keep in mind also in the future, thank you!